client/src/app/Home/home.component.ts

```diff
@@ -1,9 +1,46 @@
-import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { Component, OnInit } from '@angular/core';
 
 @Component({
   selector: 'home-app',
   templateUrl: 'app/home/home.component.html',
+  styleUrls: ['app/home/home.component.css']
```

Comment:
So you can :)

Not as simple as just using a relative path though. Apparently the component also needs to reference a module id.

Something like this:

```
@Component({
  moduleId: module.id, //Need this too
  selector: 'home-app',
  templateUrl: './home.component.html',
  styleUrls: ['./home.component.css']
})
```

See - https://angular.io/docs/ts/latest/cookbook/component-relative-paths.html